docs/samples/Microsoft.ML.Samples/Dynamic/Normalizer.cs

```diff
@@ -56,8 +56,8 @@ public static void Example()
 
             // Composing a different pipeline if we wanted to normalize more than one column at a time. 
             // Using log scale as the normalization mode. 
-            var multiColPipeline = ml.Transforms.NormalizeMinMax("LogInduced", "Induced")
-                .Append(ml.Transforms.NormalizeMinMax("LogSpontaneous", "Spontaneous"));
+            var multiColPipeline = ml.Transforms.NormalizeLogMeanVariance(new[] { new InputOutputColumnPair("LogInduced", "Induced"), new InputOutputColumnPair("LogSpontaneous", "Spontaneous") });
```

Comment:
Why the change from `NormalizeMinMax` to `NormalizeLogMeanVariance`?

Reply:
Hey Eric, it was initially LogMeanVariance but last change to this file accidentally changed to MinMax. I also verified the outputs are same as the comments. You can see output column names have "Log"

---
In reply to: [271416903](https://github.com/dotnet/machinelearning/pull/3172#discussion_r271416903) [](ancestors = 271416903)